lib/index.js

```diff
@@ -19,6 +19,7 @@ var defaults = {
   debug: identity,
   lint: true,
   minify: false,
+  encapsulateStyles: true,
```

Comment:
Sure, I was planning to make a short video tutorial also and put it on egghead.io